source/jormungandr/jormungandr/autocomplete/geocodejson.py

```diff
@@ -127,12 +127,12 @@ def get_lon_lat(obj):
     return lon, lat
 
 
-def create_address_field(geocoding):
+def create_address_field(geocoding, poi_lat, poi_lon):
```

Comment:
Why not populate `geocoding.address.coord`before (if empty) instead of adding params?
If we keep those params, default them to None might be a bit better.

Reply:
I think we parse and modify two times if i modify the code as you suggest?